test/src/unittests/machinelearning/test_tensorflowpredict.py

```diff
@@ -36,14 +34,14 @@ def regression(self, parameters):
         frameSize = 1024
         hopSize = frameSize
 
-        filename = join(testdata.audio_dir, 'recorded', 'cat_purrrr.wav')
+        filename = join(testdata.audio_dir, "recorded", "cat_purrrr.wav")
 
         audio = MonoLoader(filename=filename)()
 
-        w = Windowing(type='hann', zeroPadding=frameSize)
+        w = Windowing(type="hann", zeroPadding=frameSize)
```

Comment:
I accidentally ran my code formater (black) on the whole file and actually I prefer how it looks (maybe because I'm used to it).
I propose slowly transitioning to this (or another) standard format that we can apply consistantly.
What do you think @dbogdanov ?

Reply:
yes, let's keep it here